server/graphql/v2/interface/Transaction.js

```diff
@@ -240,5 +243,12 @@ export const TransactionFields = () => {
         return transaction; // Individual fields are set by TransactionPermissions's resolvers
       },
     },
+    virtualCardEmitterAccount: {
+      type: Account,
+      description: '',
+      async resolve(transaction) {
+        return transaction.getVirtualCardEmitterCollective();
```

Comment:
We should use a GraphQL loader for this one, transactions are usually fetched in lists so it's quite important